### PR TITLE
[5.3] Add routes helper function

### PR DESCRIPTION
**Pull request by mrjosh**
add routes_path helper function to get route directory

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
-/phpunit.xml
\ No newline at end of file
+/phpunit.xml
+.idea
```
Comment (ntzm) [inline .gitignore]: You should take this out and add `.idea` to your global gitignore!

Comment (GrahamCampbell) [inline .gitignore]: Please revert this change.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
:-1: On this from me. There is no reason to assume an app is structured like this. I know mine arn't.

**Comment by taylorotwell** [comment]
No plans on adding this.
